notebooks/COS/ViewData/ViewData.ipynb

```diff
@@ -0,0 +1,1452 @@
+    "\n",
+    "\n",
+    "# 0. Introduction\n",
+    "#### The Cosmic Origins Spectrograph ([*COS*](https://www.nasa.gov/content/hubble-space-telescope-cosmic-origins-spectrograph)) is an ultraviolet spectrograph on-board the Hubble Space Telescope ([*HST*](https://www.stsci.edu/hst/about)) with capabilities in the near ultraviolet (*NUV*) and far ultraviolet (*FUV*).\n",
+    "\n",
+    "#### This tutorial aims to prepare you to begin analyzing COS data of your choice by walking you through reading and viewing a spectrum obtained with COS, as well as obtaining a few diagnostic measurements of your spectrum.\n",
+    "\n",
+    "- For an in-depth manual to working with COS data and a discussion of caveats and user tips, see the [COS Data Handbook](https://hst-docs.stsci.edu/display/COSDHB/).\n",
+    "- For a detailed overview of the COS instrument, see the [COS Instrument Handbook](https://hst-docs.stsci.edu/display/COSIHB/).\n",
+    "\n",
+    "#### Notes for those new to Python/Jupyter/Coding:\n",
+    "\n",
+    "- You will frequently see exclamation points (**\\!**) or dollar signs (**\\$**) at the beginning of a line of code we are telling you to run. These are not part of the actual commands. The exclamation points tell a jupyter notebook to pass the following line to the command line, and the dollar sign merely indicates the start of a terminal prompt. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will import the following packages:\n",
+    "\n",
+    "- `numpy` to handle array functions (version $\\ge$ 1.17)\n",
+    "  - We also insert a line telling numpy to ignore a certain type of warning about divide-by-zeros. This is not always a good idea, so seriously consider it before copying it into your code!\n",
+    "- `astropy.io fits` for accessing FITS files\n",
+    "- `astropy.table Table` for creating tidy tables of the data\n",
+    "- `astropy.units` and `astropy.visualization.quantity_support` for dealing with units\n",
+    "- `matplotlib.pyplot` for plotting data\n",
+    "- `astroquery.mast Mast and Observations` for finding and downloading data from the [MAST](https://mast.stsci.edu/portal/Mashup/Clients/Mast/Portal.html) archive\n",
+    "\n",
+    "*Later on, we will import some functions from a local file*, for which we have imported the `sys` package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This line is just to make matplotlib look good in a notebook\n",
+    "%matplotlib inline\n",
+    "# Manipulating arrays\n",
+    "import numpy as np\n",
+    "# Our arrays will encounter divide by 0 warnings, which is OK, but we want to suppress the warning text with the next code line.\n",
+    "   # SO DO NOT COPY THIS NEXT LINE INTO YOUR CODE WITHOUT CAREFUL CONSIDERATION!\n",
+    "np.warnings.filterwarnings('ignore')\n",
```

Comment:
I suggest having this be a separate cell all by itself.  That will make it a bit more likely people won't just do what you said here anyway of blindly copying it over.